src/Microsoft.AspNetCore.WebHooks.Receivers/WebHookConstants.cs

```diff
@@ -57,6 +57,13 @@ public static class WebHookConstants
             ReceiverConfigurationSectionKey,
             "DisableHttpsCheck");
 
+        /// <summary>
+        /// Gets the model name for the root of body-bound objects. Places errors for <c>data</c> parameters in
+        /// consistent <see cref="Mvc.ModelBinding.ModelStateDictionary"/> entries, separate from entries for (for
+        /// example) parameters bound to route values.
+        /// </summary>
+        public static string ModelStateBodyModelName => "$Body";
```

Comment:
can we make this lowercase?

Reply:
Curious: MSD is case-insensitive. Why worry about the case of this key?

Reply:
because the primary use of this is to serialize it as JSON, and JSON is typically camelCase

Reply:
:ok: then